pos_restrict_users/views/pos_config_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="pos_config_view_form_users" model="ir.ui.view">
+        <field name="name">pos_config_view_form_users</field>
+        <field name="model">pos.config</field>
+        <field name="inherit_id" ref="point_of_sale.pos_config_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//sheet/div[last()]" position="after">
+                <h2 groups="point_of_sale.group_pos_manager">Users</h2>
+                <div class="row mt16 o_settings_container" groups="point_of_sale.group_pos_manager">
+                    <div class="col-12 col-lg-6 o_setting_box">
+                        <div class="o_setting_right_pane">
+                            <div class="text-muted">
+                                Users assigned to this point of sale
+                            </div>
+                            <div class="content-group">
+                                <div class="row mt16" title="Whenever you close a session, one entry is generated in the following accounting journal for all the orders not invoiced. Invoices are recorded in accounting separately.">
```

Comment:
```suggestion
                                <div class="row mt16">
```
This title doesn't make any sense with the module.
Remove it or even better, replace it with something that applies

Maybe you could explain here, that in order for the restriction to apply, the user needs the "User: Assigned POS Only"

Reply:
Thanks, done